### DISCUSSION TITLE
Compatibilty with Mojave?

### DISCUSSION THREAD

**Discussion by Jaff** [The-Wineskin-Project/WineskinServer#551] [Q&A] [answered]
I tried to install on Mojave 10.14.6) partition (I have my reasons) on MacBook Pro (Retina, 15-inch, Mid 2015) and installer said "This software does not run on macOS versions older than Catalina"
Can I generate a version backwards compatible?

**Comment by Gcenx** [comment, marked as answer]
I’m unable to support macOS Mojave in Wineskin anymore as I no longer even own an Intel Mac.

It doesn’t really take much work to support macOS Catalina as there’s prebuilt libraries available via macports, that’s not true for universal dylibs for macOS Mojave.
